src/summary/history/programs/programs-card.component.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import style from "./conditions-card-style.css";
+import dayjs from "dayjs";
+import SummaryCard from "../../cards/summary-card.component";
+import SummaryCardRow from "../../cards/summary-card-row.component";
+import SummaryCardRowContent from "../../cards/summary-card-row-content.component";
+import { match } from "react-router";
+import { performPatientProgramsSearch } from "./programs.resource";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import HorizontalLabelValue from "../../cards/horizontal-label-value.component";
+import { useCurrentPatient } from "@openmrs/esm-api";
+import SummaryCardFooter from "../../cards/summary-card-footer.component";
+import { Trans, useTranslation } from "react-i18next";
+
+export default function Programs(props: ProgramsCardProps) {
+  const [patientPrograms, setPatientPrograms] = React.useState(null);
+  const [
+    isLoadingPatient,
+    patient,
+    patientUuid,
+    patientErr
+  ] = useCurrentPatient();
+  const { t } = useTranslation();
+
+  React.useEffect(() => {
+    const subscription = performPatientProgramsSearch(patientUuid).subscribe(
+      programs => setPatientPrograms(programs),
+      createErrorHandler()
+    );
+
+    return () => subscription.unsubscribe();
+  }, [patientUuid]);
+
+  return (
+    <SummaryCard name={t("care programs", "Care Programs")} match={props.match}>
+      <SummaryCardRow>
+        <SummaryCardRowContent>
+          <HorizontalLabelValue
+            label={t("Active Programs", "Active Programs")}
+            labelStyles={{
+              color: "var(--omrs-color-ink-medium-contrast)",
+              fontFamily: "Work Sans"
+            }}
+            value={t("Since", "Since")}
+            valueStyles={{
+              color: "var(--omrs-color-ink-medium-contrast)",
+              fontFamily: "Work Sans"
+            }}
+          />
+        </SummaryCardRowContent>
+      </SummaryCardRow>
+      {patientPrograms &&
+        patientPrograms.map(program => {
+          return (
+            <SummaryCardRow key={program.uuid} linkTo="">
```

Comment:
why the empty `linkTo`?

Reply:
Because there's no level 2 yet. I will update once I create the level 2.